tests/dummy/app/router.ts

```diff
@@ -8,5 +8,6 @@ export default class Router extends EmberRouter {
 }
 Router.map(function () {
     this.route("vendors");
+	this.route("debug");
```

Comment:
I dont think this new route is necessary, just replace the index route with the component usage

Reply:
Okay, then the index template will also be replaced with the new component. Will do that in a next commit.